build

```diff
@@ -48,6 +45,8 @@ docker buildx build --platform=linux/amd64 --pull \
   --label org.opencontainers.image.ref.name="sqitch${PKG}-${VERSION}" \
   --label org.opencontainers.image.title="Sqitch" \
   --label org.opencontainers.image.description="Sensible database change management" \
+  --platform linux/amd64 \
```

Comment:
Hmm, I think this is [breaking CI](https://github.com/kolenaIO/docker-sqitch/actions/runs/4680510669/jobs/8291948630) — can we omit?

Reply:
I think so -- looks like this is already specified above now.